api-module-library/42matters/package.json

```diff
@@ -1,31 +1,28 @@
 {
-  "name": "@friggframework/api-module-42matters",
-  "version": "0.0.5",
-  "prettier": "@friggframework/prettier-config",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "lint:fix": "prettier --write --loglevel error . && eslint . --fix",
-    "test": "jest"
-  },
-  "author": "",
-  "license": "MIT",
-  "devDependencies": {
-    "@friggframework/assertions": "^1.0.21",
-    "@friggframework/eslint-config": "^1.0.20",
-    "@friggframework/logs": "^1.0.23",
-    "@friggframework/module-plugin": "^1.1.15",
-    "@friggframework/test-environment": "^1.1.20",
-    "dotenv": "^16.3.1",
-    "eslint": "^8.49.0",
-    "jest": "^29.7.0",
-    "prettier": "^3.0.3",
-    "sinon": "^16.0.0"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
-  "dependencies": {
-    "md5": "^2.3.0"
-  }
-}
+    "name": "@friggframework/api-module-42matters",
+    "version": "0.0.5",
+    "prettier": "@friggframework/prettier-config",
```

Comment:
Is this now `@friggframework/devtools/prettier-config` or should we release a prettier-config?

Reply:
oh hmm, interesting. i don't know if `@friggframework/devtools/prettier-config` would work, since it's exported at the root of devtools and isn't a separate package
`@friggframework/devtools` would be enough here?